cmd/kubeadm/app/phases/certs/certlist.go

```diff
@@ -397,3 +397,68 @@ func setCommonNameToNodeName() configMutatorsFunc {
 		return nil
 	}
 }
+
+// leafCertificates returns non-CA certificates from the supplied Certificates.
+func leafCertificates(c Certificates) (Certificates, error) {
+	certTree, err := c.AsMap().CertTree()
+	if err != nil {
+		return nil, errors.WithStack(err)
```

Comment:
we should just return `err` or errors.Wrap(f) in instead of WithStack (applies to the rest of the diff too)